`fake-api` example does not compile

Running `trunk serve` results in compilation errors

```
error[E0432]: unresolved import `crate::generic_node::DomType`
 --> crates/hirola-core/src/dom.rs:2:20
  |
2 |     generic_node::{DomType, GenericNode},
  |                    ^^^^^^^ no `DomType` in `generic_node`

error[E0432]: unresolved import `crate::generic_node::DomType`
 --> crates/hirola-core/src/render.rs:4:20
  |
4 |     generic_node::{DomType, GenericNode},
  |                    ^^^^^^^ no `DomType` in `generic_node`

error[E0432]: unresolved import `crate::generic_node::DomType`
 --> crates/hirola-core/src/templating/flow.rs:6:27
  |
6 | use crate::generic_node::{DomType, GenericNode};
  |                           ^^^^^^^ no `DomType` in `generic_node`

error[E0432]: unresolved import `crate::generic_node::DomType`
 --> crates/hirola-core/src/templating/noderef.rs:2:5
  |
2 | use crate::generic_node::DomType;
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `DomType` in `generic_node`

error[E0432]: unresolved import `crate::generic_node::DomType`
 --> crates/hirola-core/src/templating/suspense.rs:3:20
  |
3 |     generic_node::{DomType, GenericNode},
  |                    ^^^^^^^ no `DomType` in `generic_node`

error[E0432]: unresolved import `crate::generic_node::DomType`
 --> crates/hirola-core/src/templating/switch.rs:7:20
  |
7 |     generic_node::{DomType, GenericNode},
  |                    ^^^^^^^ no `DomType` in `generic_node`

error[E0412]: cannot find type `Event` in this scope
  --> crates/hirola-core/src/generic_node.rs:22:33
   |
22 | pub type EventListener = dyn Fn(Event);
   |                                 ^^^^^ not found in this scope

warning: unused import: `crate::dom::*`
  --> crates/hirola-core/src/lib.rs:36:5
   |
36 | use crate::dom::*;
   |     ^^^^^^^^^^^^^
   |
   = note: `#[warn(unused_imports)]` on by default

    Checking gloo-net v0.1.0
error[E0308]: mismatched types
   --> crates/hirola-core/src/dom.rs:128:33
    |
128 |     pub fn inner_html(&self) -> String {
    |            ----------           ^^^^^^ expected `String`, found `()`
    |            |
    |            implicitly returns `()` as its body has no tail or `return` expression

warning: unused import: `GenericNode`
 --> crates/hirola-core/src/dom.rs:2:29
  |
2 |     generic_node::{DomType, GenericNode},
  |                             ^^^^^^^^^^^

warning: unused import: `GenericNode`
 --> crates/hirola-core/src/render.rs:4:29
  |
4 |     generic_node::{DomType, GenericNode},
  |                             ^^^^^^^^^^^

warning: unused import: `GenericNode`
 --> crates/hirola-core/src/templating/flow.rs:6:36
  |
6 | use crate::generic_node::{DomType, GenericNode};
  |                                    ^^^^^^^^^^^

warning: unused import: `GenericNode`
 --> crates/hirola-core/src/templating/switch.rs:7:29
  |
7 |     generic_node::{DomType, GenericNode},
  |                             ^^^^^^^^^^^

warning: unused import: `GenericNode`
 --> crates/hirola-core/src/templating/suspense.rs:3:29
  |
3 |     generic_node::{DomType, GenericNode},
  |                             ^^^^^^^^^^^

Some errors have detailed explanations: E0308, E0412, E0432.
For more information about an error, try `rustc --explain E0308`.
warning: `hirola-core` (lib) generated 6 warnings
error: could not compile `hirola-core` (lib) due to 8 previous errors; 6 warnings emitted
warning: build failed, waiting for other jobs to finish...
```

I suspect the example is missing the `"dom"` feature perhaps. From my testing locally, adding this feature fixes this issue. Should I open a PR for this?

--- a/examples/fake-api/Cargo.toml
+++ b/examples/fake-api/Cargo.toml
@@ -7,7 +7,7 @@ edition = "2021"
 
 [dependencies]
 anyhow = "1"
-hirola = { path = "../../" }
+hirola = { path = "../../", features = ["dom"] }
 serde = { version = "1.0.80", features = ["derive"] }
 reqwasm = "0.5.0"
 

